tests/sentry/interfaces/test_message.py

```diff
@@ -19,6 +19,13 @@ def interface(self):
             )
         )
 
+    def test_null_values(self):
+        sink = {}
+        assert Message.to_python({}).to_json() == sink
+        assert Message.to_python({'message': None}).to_json() == sink
+        assert Message.to_python({'formatted': None}).to_json() == sink
+        assert Message.to_python({'params': None}).to_json() == sink
```

Comment:
Do we know if downstream consumers of the interface types will be tolerant of previously required keys being missing?

Reply:
we're still trying to figure this out. We're testing this on garbage right now with mixed results